### ISSUE TITLE
Support indexing of dataset publisher organizationName

### ISSUE THREAD

**Issue by csjx** [NCEAS/metacat#1255]
In MetacatUI, we plan to use the EML `/eml/dataset/publisher/organizationName` in the citation for certain MNs, such as ESS-DIVE. We currently don't index this field in Solr, and so it's not available in MetacatUI.  Change the indexer rules in Metacat (and ideally upstream in `d1_index_processor`) to index this field. We may also consider indexing the other `eml-party` fields unde `/eml/dataset/publisher`.

**Comment by gothub** [comment]
hey @csjx @taojing2002 I self-assigned this one. I've worked on the indexing component before.

**Comment by csjx** [comment]
Sounds good @gothub !

**Comment by jeanetteclark** [comment]
bump? @mbjones

**Comment by mbjones** [comment]
Agreed. @taojing2002 can you add this to the list of indexing changes for Metacat?

**Comment by taojing2002** [comment]
Sure. I will do it next week after I finish the osti library.
